test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -0,0 +1,76 @@
+Describe "Job Cmdlet Tests" -Tag "CI" {
```

Comment:
thanks Jim, that sounds reasonable. BTW, what does `-pending` mean in Pester tests?